websites.yml

```diff
@@ -46,6 +48,19 @@ username_patterns:
     invalid_patterns:
       - "\\.(com|net)"
       - "(\\.)\\1{1,}"  # consecutive '.' not allowed
+  openhub:
+    characters: a-zA-Z0-9_-
+    min_length: 3
+    max_length: 40
+    invalid_patterns:
+      - "^[^a-zA-Z]"
+  deviantart:
```

Comment:
also add these keys in alpha order, but dont re-order the existing ones.

i.e. do 'best effort' or make your changes roughly ordered correctly.  Later we'll re-order the other ones.

Reply:
hmm... what's 'best effort'? what do you mean "make your changes roughly ordered correctly"? @jayvdb

Reply:
and what key should I sort?

Reply:
`deviantart` should be after `behance`

Reply:
got it ;-)